.github/workflows/ci.yml

```diff
@@ -4,18 +4,25 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
-
+    runs-on: ubuntu-24.04
     steps:
-    - uses: actions/checkout@v3
-    - uses: actions/cache@v3
+    - uses: actions/checkout@v4
+    - name: Install OpenSSL 1.1.1 (for OTP23/24)
```

Comment:
OTP 24 doesn't work with OpenSSL 3?

We could almost drop OTP 23-24, but I guess we can keep this now since it's not too many lines.

Reply:
I seemed not to build out of the box when I tested.
To be able to lift erlide to support OTP25-27 I think I need to drop 23-24 though (and openssl 1.1.1 can be removed again).
We will then use OTP 25 when building erlide_kernel and OTP 26/27 for the debugger support. rebar3, which is used in the build/scripts, also only support a 3 version range. I don't really want to fix any build scripts and redo how the build is done, so having the same limitation simplifies a bit.